CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -208,7 +229,7 @@ local function EquipNewTool(tool) --NOTE: HopperBin
 end
 
 local function IsEquipped(tool)
-	return tool and ((tool:IsA('HopperBin') and tool.Active) or tool.Parent == Character) --NOTE: HopperBin
+	return not not (tool and ((tool:IsA('HopperBin') and tool.Active) or tool.Parent == Character)) --NOTE: HopperBin
```

Comment:
This is a no-no, if you want to convert to a bool for nil object checks please use:
`local myBool = (object ~= nil)`